openhands/core/config/condenser_config.py

```diff
@@ -43,6 +43,14 @@ class LLMSummarizingCondenserConfig(BaseModel):
     llm_config: LLMConfig = Field(
         ..., description='Configuration for the LLM to use for condensing.'
     )
+    keep_first: int = Field(
+        default=0,
```

Comment:
```suggestion
        default=1,
```

Maybe we can default to at least "the task"